dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/deployments/MergeConfigServerXMLTest.java

```diff
@@ -63,18 +68,295 @@ public class MergeConfigServerXMLTest {
 
     @BeforeClass
     public static void startup() throws Exception {
+        server.saveServerConfiguration();
         server.startServer();
     }
 
     @AfterClass
     public static void shutdown() throws Exception {
-        server.stopServer("CWWKO1683W"); // Invalid info element
+        server.stopServer("CWWKO1683W" // Invalid info element
```

Comment:
The comment is inside the method parentheses.

Reply:
Yeah, this was deliberate because I meant to add more messages here and each message wants its own comment.

Reply:
And now I see more messages, with more of them it makes sense.